src/status_im/contexts/wallet/send/select_asset/view.cljs

```diff
@@ -17,22 +17,21 @@
    {:id :tab/collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}])
 
 (defn- asset-component
-  []
-  (fn [token _ _ {:keys [currency currency-symbol]}]
-    (let [on-press         #(rf/dispatch [:wallet/send-select-token
-                                          {:token    token
-                                           :stack-id :wallet-select-asset}])
-          token-units      (utils/total-token-units-in-all-chains token)
-          crypto-formatted (utils/get-standard-crypto-format token token-units)
-          fiat-value       (utils/total-token-fiat-value currency token)
-          fiat-formatted   (utils/get-standard-fiat-format crypto-formatted currency-symbol fiat-value)]
-      [quo/token-network
-       {:token       (:symbol token)
-        :label       (:name token)
-        :token-value (str crypto-formatted " " (:symbol token))
-        :fiat-value  fiat-formatted
-        :networks    (:networks token)
-        :on-press    on-press}])))
+  [token _ _ {:keys [currency currency-symbol]}]
+  (let [on-press         #(rf/dispatch [:wallet/send-select-token
+                                        {:token    token
+                                         :stack-id :wallet-select-asset}])
+        token-units      (utils/total-token-units-in-all-chains token)
```

Comment:
Not from your PR, but I wonder if we are putting too much computation on the view layer. The fact that we hide them inside `wallet.common.utils` doesn't change anything. Ideally, we should design the views to use subscriptions and benefit from caching. It may not be a trivial refactor because best would be to not use subscriptions inside the component rendered by `flat-list`.

Reply:
Regarding to performance, I haven't  noticed any problem, even in users with many tokens.

Actually, recently I compared the performance (as a regular user) with other flat-list we use (such as channel list in a community or the contacts list) and this one behaves really good.

Reply:
I wasn't saying it's a problem now, good to hear it's good! o/

Theoretically we could get better performance with subscriptions, but it's not always true because comparing large data structures to get a cache hit isn't cheap, and our mobile client passes a ton of data around.

This is yet another thing we could all design better, subscriptions shouldn't return too much, ideally only the minimum the views need. Many times we subscribe to large data structures when we only need a few values, this is quite bad.